docs/operation/bootstrap_backup_restore.rst

```diff
@@ -41,9 +35,9 @@ First mount the ISO and then run the restore script:
 
 .. code::
 
-    /srv/scality/metalk8s-X.X.X/restore.sh --backup-file <backup_archive>
+   /srv/scality/metalk8s-X.X.X/restore.sh --backup-file <backup_archive> --apiserver-node-ip <node_ip>
 
 .. note::
 
     Replace `<backup_archive>` with the path to the backup archive you want
-    to use.
+    to use and `<node_ip>` with a node IP of one apiserver.
```

Comment:
Maybe mention "control-plane IP" and refer to simpler concepts (control-plane Node instead of `apiserver`, or at least refer to the Glossary).